tests/integration/test_db.py

```diff
@@ -233,7 +233,7 @@ async def test_discourse(ops_test: OpsTest):
     # Deploy Discourse and Redis.
     await gather(
         ops_test.model.deploy(DISCOURSE_APP_NAME, application_name=DISCOURSE_APP_NAME),
-        ops_test.model.deploy(REDIS_APP_NAME, application_name=REDIS_APP_NAME, channel="edge"),
+        ops_test.model.deploy(REDIS_APP_NAME, application_name=REDIS_APP_NAME),
```

Comment:
Edge seems to consistently error out on 1.28.

Reply:
> Should we report it? Otherwise one day their edge will be stable => our troubles here.

It's one of our charms. It deploys fine on it's own on 1.28. I'll try to collect more details later.